### PR TITLE
added Drum Sound Project

### PR DESCRIPTION
**Pull request by Nayan808**
Hello Project Admin,
## Screenshot -
Screenshot from 2023-06-25 15-08-55

# Checklist:

- [x] I have mentioned the issue number in my Pull Request.
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`

### CODE REVIEW

**Review by MohitGupta121 — CHANGES_REQUESTED** [review]
@Nayan808   please remove all the images or gif from your PR and use them through cloud link. Upload somewhere the images and use them from that link.

File: Drum-Sounds/.DS_Store
Comment (MohitGupta121) [inline Drum-Sounds/.DS_Store:1]: Also remove this file

**Review by MohitGupta121 — CHANGES_REQUESTED** [review]

**Review by Nayan808 — COMMENTED** [review]
@MohitGupta121 
Edited the Code And Added Images By cloud And Removed files

File: Drum-Sounds/.DS_Store
Comment (MohitGupta121) [inline Drum-Sounds/.DS_Store:1]: @Nayan808 Also remove this file from PR

File: Drum-Sounds/Readme.md
Comment (MohitGupta121) [inline Drum-Sounds/Readme.md:1]: @Nayan808 why readme is empty?

**Review by MohitGupta121 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**